.extensions/drupal/extension.py

```diff
@@ -12,7 +12,7 @@
 
 
 DEFAULTS = utils.FormattedDict({
-    'DRUPAL_VERSION': '7.37',
+    'DRUPAL_VERSION': '7.53',
     'DRUPAL_PACKAGE': 'drupal-{DRUPAL_VERSION}.tar.gz',
     'DRUPAL_HASH': 'b008183acc8c2e6fca13755602e1dc9035fdbfc2',
```

Comment:
You need to update the hash too.  It's a simple md5 of the binary to be downloaded.